script.js

```diff
@@ -0,0 +1,203 @@
+    if (element.length === 1) {
+      element = element[0];
+      element.classList.add('keypress');
+      switch (element.innerText) {
+        case 'Backspace':
+          output = output.substr(0, output.length-1);
+          break;
+        case 'Enter':
+          output += '\n';
+          break;
+        case '':
+          output += ' ';
+          break;
+        case 'Tab':
+          output += '\t';
+          break;
+        case 'Caps Lock':
+          output += '';
+          break;
+        default:
+        output += element.innerText;
+      }
+      document.querySelector('textarea').value = output;
+    } else {
+      specialKeysFunctionAdd(element, event.code);
+    }
+    if (event.keyCode === 16) { //Shift
+      let langKeys = isEng === '1' ? engKeysShift : rusKeysShift;
+      keysContent(langKeys);
+    }
+  }
+
+  document.addEventListener('keydown', (event) => {
+    keydown(event);
+  });
+  document.addEventListener('mousedown', (e) => {
+    let element = e.target;
+    if (element.parentNode != document.getElementById('keyboard')) return;
+    element.classList.add('keypress');
+  });
+  document.addEventListener('mouseup', (e) => {
+    let element = e.target;
+    if (element.parentNode != document.getElementById('keyboard')) return;
+    element.classList.remove('keypress');
+    switch (element.innerText) {
+      case 'Backspace':
+        output = output.substr(0, output.length-1);
+        break;
+      case 'Enter':
+        output += '\n';
+        break;
+      case '':
+        output += ' ';
+        break;
+      case 'Tab':
+        output += '\t';
+        break;
+      case 'Caps Lock':
+        output += '';
+        break;
+      case 'Win':
+        return;
+      case 'Alt':
+        return;
+      case 'Shift':
+        return;
+      case 'Ctrl':
+        return;
+      default:
+      output += element.innerText;
+    }
+    document.querySelector('textarea').value = output;
+    let keyCode = event.target.getAttribute('data');
+    if (keyCode === 20) { //Caps Lock
+      capsLock = !capsLock;
+      let langKeys = isEng === 1 ? engKeysUpperCase : rusKeysUpperCase;
+
+      if (capsLock) {
+        element.classList.add('keypress');
+        keysContent(langKeys);
+      } else {
+        langKeys = isEng === '1' ? engKeys : rusKeys;
+        element.classList.remove('keypress');
+        keysContent(langKeys);
+      }
+    }
+  });
+  document.addEventListener('keyup', function(event) {
+    if (event.code === 'ShiftLeft') {
+      leftShift = false;
+      if (leftAlt) {
+        isEng = isEng === '1' ? '0' : '1';
+        languageHtml.innerText = isEng === '1' ? 'Eng' : 'Rus';
+        let langKeys;
+        isEng === '1' ? langKeys = engKeys : langKeys = rusKeys;
+        keysContent(langKeys);
+      }
+    } else if (event.code === 'AltLeft') {
+      leftAlt = false;
+      if (leftShift) {
+        isEng = isEng === '1' ? '0' : '1';
```

Comment:
Update it const isEng = Number(!isEng);
And why do you keep in localstorage the string '1' save it as a number

Reply:
isEng now is Number. Can't use const because isEng is changing.